support/android/apk/build.xml

```diff
@@ -36,9 +36,17 @@
     <condition property="sdk.dir" value="${env.ANDROID_HOME}">
         <isset property="env.ANDROID_HOME" />
     </condition>
-    <condition property="target" value="${env.ANDROID_PLATFORM}">
-        <isset property="env.ANDROID_PLATFORM" />
-    </condition>
+
+    <!-- The project.properties file is created and updated by the 'android'
+         tool, as well as ADT.
+
+         This contains project specific properties such as project target, and library
+         dependencies. Lower level build properties are stored in ant.properties
+         (or in .classpath for Eclipse projects).
+
+         This file is an integral part of the build system for your
+         application and should be checked into Version Control Systems. -->
+    <loadproperties srcFile="project.properties" />
```

Comment:
I believe @mitchhentges removed this file in #11278. If the Android automated tooling rewrote the build.xml file to link to it, then we should re-add the project.properties file. If you changed build.xml by hand, I think we can revert that change.